scripts/gdb/gdbinit.py

```diff
@@ -0,0 +1,12 @@
+import sys
+from os import path
+
+here = path.dirname(path.abspath(__file__))
+
+if __name__ == "__main__":
+    if here not in sys.path:
+        sys.path.insert(0, here)
+
+    [sys.modules.pop(key) for key in filter(lambda m: m.startswith("lvglgdb"), sys.modules.keys())]
```

Comment:
It's not that readable especially the generated list is not used.
Please consider
for key in xxx:
   sys.modules.pop(key)